Custom compiling error

Cant figure out whats happening...
`   Compiling facade v0.1.0 (https://github.com/DenisKolodin/facade#0498b46c)
error: failed to run custom build command for `facade v0.1.0 (https://github.com/DenisKolodin/facade#0498b46c)`

Caused by:
  process didn't exit successfully: `c:\dev\capitalist_bot_v5\target\debug\build\facade-b9a32e24080021bf\build-script-build` (exit code: 1)
--- stderr
   Compiling facade-ui v0.1.0 (C:\Users\serzh\.cargo\git\checkouts\facade-ce9b2e419f186bdf\0498b46\ui)
error[E0432]: unresolved import `yew::Properties`
 --> src\widgets\bind.rs:2:17
  |
2 | use yew::{html, Properties};
  |                 ^^^^^^^^^^ no `Properties` in the root

error[E0432]: unresolved import `yew::Properties`
 --> src\widgets\container.rs:3:17
  |
3 | use yew::{html, Properties};
  |                 ^^^^^^^^^^ no `Properties` in the root

error[E0432]: unresolved import `yew::Properties`
 --> src\widgets\control.rs:2:17
  |
2 | use yew::{html, Properties};
  |                 ^^^^^^^^^^ no `Properties` in the root

error[E0432]: unresolved import `yew::Properties`
 --> src\widgets\dashboard.rs:3:17
  |
3 | use yew::{html, Properties, ShouldRender};
  |                 ^^^^^^^^^^ no `Properties` in the root

error[E0432]: unresolved import `yew::Properties`
 --> src\widgets\dynamic.rs:4:17
  |
4 | use yew::{html, Properties, ShouldRender};
  |                 ^^^^^^^^^^ no `Properties` in the root

error[E0432]: unresolved import `yew::Properties`
 --> src\widgets\fixed.rs:3:17
  |
3 | use yew::{html, Properties};
  |                 ^^^^^^^^^^ no `Properties` in the root

error[E0432]: unresolved import `yew::Properties`
 --> src\widgets\layout.rs:3:17
  |
3 | use yew::{html, Properties};
  |                 ^^^^^^^^^^ no `Properties` in the root

error[E0432]: unresolved import `yew::Properties`
 --> src\widgets\list.rs:2:17
  |
2 | use yew::{html, Properties};
  |                 ^^^^^^^^^^ no `Properties` in the root

error[E0432]: unresolved import `yew::Properties`
 --> src\widgets\page.rs:3:17
  |
3 | use yew::{html, Properties};
  |                 ^^^^^^^^^^ no `Properties` in the root

error[E0432]: unresolved import `yew::Properties`
 --> src\widgets\panel.rs:2:17
  |
2 | use yew::{html, Properties};
  |                 ^^^^^^^^^^ no `Properties` in the root

error[E0432]: unresolved import `yew::Properties`
 --> src\widgets\scene.rs:4:17
  |
4 | use yew::{html, Properties, ShouldRender};
  |                 ^^^^^^^^^^ no `Properties` in the root

error[E0432]: unresolved import `yew::Properties`
 --> src\widgets\widget.rs:3:60
  |
3 | use yew::{Bridge, Bridged, Component, ComponentLink, Html, Properties, Renderable, ShouldRender};
  |                                                            ^^^^^^^^^^ no `Properties` in the root

error: cannot determine resolution for the derive macro `Properties`
  --> src\widgets\bind.rs:10:10
   |
10 | #[derive(Properties, PartialEq, Clone)]
   |          ^^^^^^^^^^
   |
   = note: import resolution is stuck, try simplifying macro imports

error: cannot determine resolution for the derive macro `Properties`
  --> src\widgets\container.rs:11:10
   |
11 | #[derive(Properties, PartialEq, Clone)]
   |          ^^^^^^^^^^
   |
   = note: import resolution is stuck, try simplifying macro imports

error: cannot determine resolution for the derive macro `Properties`
  --> src\widgets\control.rs:10:10
   |
10 | #[derive(Properties, PartialEq, Clone)]
   |          ^^^^^^^^^^
   |
   = note: import resolution is stuck, try simplifying macro imports

error: cannot determine resolution for the derive macro `Properties`
  --> src\widgets\dashboard.rs:12:10
   |
12 | #[derive(Properties, PartialEq, Clone)]
   |          ^^^^^^^^^^
   |
   = note: import resolution is stuck, try simplifying macro imports

error: cannot determine resolution for the derive macro `Properties`
  --> src\widgets\dynamic.rs:12:10
   |
12 | #[derive(Properties, PartialEq, Clone)]
   |          ^^^^^^^^^^
   |
   = note: import resolution is stuck, try simplifying macro imports

error: cannot determine resolution for the derive macro `Properties`
  --> src\widgets\fixed.rs:11:10
   |
11 | #[derive(Properties, PartialEq, Clone)]
   |          ^^^^^^^^^^
   |
   = note: import resolution is stuck, try simplifying macro imports

error: cannot determine resolution for the derive macro `Properties`
  --> src\widgets\layout.rs:11:10
   |
11 | #[derive(Properties, PartialEq, Clone)]
   |          ^^^^^^^^^^
   |
   = note: import resolution is stuck, try simplifying macro imports

error: cannot determine resolution for the derive macro `Properties`
  --> src\widgets\list.rs:10:10
   |
10 | #[derive(Properties, PartialEq, Clone)]
   |          ^^^^^^^^^^
   |
   = note: import resolution is stuck, try simplifying macro imports

error: cannot determine resolution for the derive macro `Properties`
  --> src\widgets\page.rs:11:10
   |
11 | #[derive(Properties, PartialEq, Clone)]
   |          ^^^^^^^^^^
   |
   = note: import resolution is stuck, try simplifying macro imports

error: cannot determine resolution for the derive macro `Properties`
  --> src\widgets\panel.rs:10:10
   |
10 | #[derive(Properties, PartialEq, Clone)]
   |          ^^^^^^^^^^
   |
   = note: import resolution is stuck, try simplifying macro imports

error: cannot determine resolution for the derive macro `Properties`
  --> src\widgets\scene.rs:12:10
   |
12 | #[derive(Properties, PartialEq, Clone)]
   |          ^^^^^^^^^^
   |
   = note: import resolution is stuck, try simplifying macro imports

error[E0658]: the attribute `props` is currently unknown to the compiler and may have meaning added to it in the future
  --> src\widgets\bind.rs:12:5
   |
12 |     #[props(required)]
   |     ^^^^^^^^^^^^^^^^^^
   |
   = note: for more information, see https://github.com/rust-lang/rust/issues/29642
   = help: add `#![feature(custom_attribute)]` to the crate attributes to enable

error[E0658]: the attribute `props` is currently unknown to the compiler and may have meaning added to it in the future
  --> src\widgets\container.rs:13:5
   |
13 |     #[props(required)]
   |     ^^^^^^^^^^^^^^^^^^
   |
   = note: for more information, see https://github.com/rust-lang/rust/issues/29642
   = help: add `#![feature(custom_attribute)]` to the crate attributes to enable

error[E0658]: the attribute `props` is currently unknown to the compiler and may have meaning added to it in the future
  --> src\widgets\control.rs:12:5
   |
12 |     #[props(required)]
   |     ^^^^^^^^^^^^^^^^^^
   |
   = note: for more information, see https://github.com/rust-lang/rust/issues/29642
   = help: add `#![feature(custom_attribute)]` to the crate attributes to enable

error[E0658]: the attribute `props` is currently unknown to the compiler and may have meaning added to it in the future
  --> src\widgets\dashboard.rs:14:5
   |
14 |     #[props(required)]
   |     ^^^^^^^^^^^^^^^^^^
   |
   = note: for more information, see https://github.com/rust-lang/rust/issues/29642
   = help: add `#![feature(custom_attribute)]` to the crate attributes to enable

error[E0658]: the attribute `props` is currently unknown to the compiler and may have meaning added to it in the future
  --> src\widgets\layout.rs:13:5
   |
13 |     #[props(required)]
   |     ^^^^^^^^^^^^^^^^^^
   |
   = note: for more information, see https://github.com/rust-lang/rust/issues/29642
   = help: add `#![feature(custom_attribute)]` to the crate attributes to enable

error[E0658]: the attribute `props` is currently unknown to the compiler and may have meaning added to it in the future
  --> src\widgets\list.rs:12:5
   |
12 |     #[props(required)]
   |     ^^^^^^^^^^^^^^^^^^
   |
   = note: for more information, see https://github.com/rust-lang/rust/issues/29642
   = help: add `#![feature(custom_attribute)]` to the crate attributes to enable

error[E0658]: the attribute `props` is currently unknown to the compiler and may have meaning added to it in the future
  --> src\widgets\page.rs:13:5
   |
13 |     #[props(required)]
   |     ^^^^^^^^^^^^^^^^^^
   |
   = note: for more information, see https://github.com/rust-lang/rust/issues/29642
   = help: add `#![feature(custom_attribute)]` to the crate attributes to enable

error[E0658]: the attribute `props` is currently unknown to the compiler and may have meaning added to it in the future
  --> src\widgets\panel.rs:12:5
   |
12 |     #[props(required)]
   |     ^^^^^^^^^^^^^^^^^^
   |
   = note: for more information, see https://github.com/rust-lang/rust/issues/29642
   = help: add `#![feature(custom_attribute)]` to the crate attributes to enable

error: aborting due to 31 previous errors
Some errors have detailed explanations: E0432, E0658.
For more information about an error, try `rustc --explain E0432`.
error: Could not compile `facade-ui`.

To learn more, run the command again with --verbose.
error: build failed

Error: ErrorMessage { msg: "Can\'t compile UI" }`

--- a/build.rs
+++ b/build.rs
@@ -1,7 +1,8 @@
 use failure::{format_err, Error};
 use std::env;
+use std::io::{BufRead, BufReader, Read};
 use std::path::PathBuf;
-use std::process::Command;
+use std::process::{Command, Stdio};
 
 trait RunIt {
     fn run_it(&mut self, err: &str) -> Result<(), Error>;
@@ -9,10 +10,20 @@ trait RunIt {
 
 impl RunIt for Command {
     fn run_it(&mut self, err: &str) -> Result<(), Error> {
-        let output = self.output()?;
-        if !output.status.success() {
-            let out = String::from_utf8_lossy(&output.stderr);
-            eprintln!("{}", out);
+        self.stderr(Stdio::piped());
+        let mut child = self.spawn()?;
+        if let Some(out) = child.stdout.take() {
+            let buf = BufReader::new(out);
+            for line in buf.lines() {
+                print!("{}", line?);
+            }
+        }
+        if !child.wait()?.success() {
+            if let Some(mut err) = child.stderr.take() {
+                let mut out = String::new();
+                err.read_to_string(&mut out)?;
+                eprintln!("{}", out);
+            }
             Err(format_err!("{}", err))
         } else {
             Ok(())

